api/controllers/transactions.js

```diff
@@ -54,60 +70,56 @@ TransactionsController.getTransactions = function(context, next) {
 	const params = context.request.swagger.params;
 
 	let filters = {
-		senderIdOrRecipientId: params.senderIdOrRecipientId.value,
 		id: params.id.value,
 		blockId: params.blockId.value,
 		recipientId: params.recipientId.value,
 		recipientPublicKey: params.recipientPublicKey.value,
 		senderId: params.senderId.value,
 		senderPublicKey: params.senderPublicKey.value,
 		type: params.type.value,
-		fromHeight: params.height.value,
```

Comment:
Why is this filter removed?

Reply:
Just mapped the param to direct filter of `height`